src/node/desktop/binding.gyp

```diff
@@ -0,0 +1,26 @@
+{
+    "targets": [
+        {
+            "target_name": "desktop",
+            "cflags!": [ "-fno-exceptions" ],
+            "cflags_cc!": [ "-fno-exceptions" ],
+            "sources": [ "src/native/desktop.cc" ],
+            "include_dirs": [
+                "<!@(node -p \"require('node-addon-api').include\")"
+            ],
+            "defines": [ "NAPI_DISABLE_CPP_EXCEPTIONS" ]
+        },
+        {
+            "target_name": "copy",
```

Comment:
This target copies the native node module back into the source tree, which makes it easier to reference in sources (and allows us to provide an accompanying `.d.ts` file for types)